neo/Ledger/MemoryPool.cs

```diff
@@ -16,32 +16,33 @@ public class MemoryPool : IReadOnlyCollection<Transaction>
     {
         private class PoolItem : IComparable<PoolItem>
         {
-            public readonly Transaction Transaction;
+            public readonly Transaction Tx;
```

Comment:
Nice change! much more readable... I was wondering if it was a class or object, now it's clear.

Reply:
kkkkkkkkk, the same as me, broda.